<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExifSort</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="pathFormatLabel">
         <property name="text">
          <string>Path Format:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Replace Whitespace with:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Attribute Selector</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="pathFormatLineEdit"/>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="whiteSpaceReplacementSelector"/>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="attributeSelectorInput"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="1">
        <widget class="QLineEdit" name="outputPathEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rootPathLabel">
         <property name="text">
          <string>Input Path:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="seperateOutputPathCheckbox">
         <property name="text">
          <string>Separate output path</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="inputPathEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Output Path:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="inputPathBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="outputPathBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QCheckBox" name="tiffCheckBox">
         <property name="text">
          <string>Tiff</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="jpgCheckBox">
         <property name="text">
          <string>JPG</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="rawCheckBox">
         <property name="text">
          <string>RAW</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QDialogButtonBox" name="mainButtonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>inputPathEdit</tabstop>
  <tabstop>inputPathBrowseButton</tabstop>
  <tabstop>seperateOutputPathCheckbox</tabstop>
  <tabstop>outputPathEdit</tabstop>
  <tabstop>outputPathBrowseButton</tabstop>
  <tabstop>jpgCheckBox</tabstop>
  <tabstop>rawCheckBox</tabstop>
  <tabstop>tiffCheckBox</tabstop>
  <tabstop>pathFormatLineEdit</tabstop>
  <tabstop>attributeSelectorInput</tabstop>
  <tabstop>whiteSpaceReplacementSelector</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
